### DISCUSSION TITLE
Microsoft Teams Integration

### DISCUSSION THREAD

**Discussion by ryanbuening** [datalust/seq-tickets#1310] [Help] [answered]
I was wondering if there is currently a way to connect Seq to a Microsoft Teams Team/Channel? The use case being to alert a group of people when certain events (errors, etc.) occur.

**Comment by augustoproiete** [comment, marked as answer]
You might want to give [Seq.App.Teams](https://github.com/AntoineGa/Seq.App.Teams) a try.
